modules/microshift-ingress-controller-config.adoc

```diff
@@ -57,11 +71,31 @@ ingress:
 # ...
 ----
 +
-.Ingress controller configuration fields definitions table
+.Ingress controller operations and performance configuration fields definitions
 [cols="3a,8a",options="header"]
 |===
 |Parameter |Description
 
+|`certificateSecret`
+|The `certificateSecret` value is a reference to a secret that contains the default certificate that is served by the Ingress Controller. When Routes do not specify their own certificate, `certificateSecret` is used.atehe Ingress Controller.
+
+For information about creating a secret, see https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/security_and_compliance/configuring-certificates#replacing-default-ingress_replacing-default-ingress [Replacing the default ingress certificate].
```

Comment:
this doc is for openshift, the main difference is the namespace name,
we dont use  `openshift-config` namespace, we use `openshift-ingress`  , so the secret has to be created there.

Reply:
resolved